src/compiler/utilities.ts

```diff
@@ -2272,7 +2272,7 @@ namespace ts {
                 sourceMapFilePath: getSourceMapFilePath(jsFilePath, options),
                 declarationFilePath
             };
-            action(emitFileNames, [sourceFile], /*isBundledEmit*/false);
+            action(emitFileNames, [sourceFile], /*isBundledEmit*/false, emitOnlyDtsFiles);

```

Comment:
It is actually, because otherwise the emitter would not write the triple slashes in the declaration file